assets/src/edit-story/components/rangeInput/index.js

```diff
@@ -30,14 +30,19 @@ const rangeThumb = css`
   appearance: none;
   width: ${({ thumbSize = 16 }) => thumbSize}px;
   height: ${({ thumbSize = 16 }) => thumbSize}px;
-  background-color: ${({ theme }) => theme.colors.fg.v1};
+  background-color: ${({ theme }) => theme.colors.fg.primary};
   cursor: pointer;
   border-radius: 50px;
   box-shadow: 0 2px 5px rgba(0, 0, 0, 0.15);
 `;
 
 const focusedRangeThumb = css`
-  background-color: ${({ theme }) => theme.colors.action};
+  border: 2px solid #8ab4f8;
```

Comment:
Does this color have a theme value?

Reply:
Fixed in bd83ceeaa. It's the [secondary accent color, _"Biru"_](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Book?node-id=1286%3A457075).